src/components/panel/panel.js

```diff
@@ -126,6 +126,16 @@ angular
  *   - `animation` - `{MdPanelAnimation=}`: An MdPanelAnimation object that
  *     specifies the animation of the panel. For more information, see
  *     `MdPanelAnimation`.
+ *   - `onDomAdded` - `{function=}`: Callback function used to announce when
+ *     the panel is added to the DOM. This function needs to return a promise
```

Comment:
We shouldn't require the user to return a promise. We can wrap their function response in one so chaining will still work on our end.